test/server/wtr-config.test.js

```diff
@@ -0,0 +1,120 @@
+import { createConfig, getBrowsers, WTRConfig } from '../../src/server/wtr-config.js';
+import { createConfig as createConfigPublic, getBrowsers as getBrowsersPublic } from '../../src/index.js';
+import { expect } from 'chai';
+import sinon from 'sinon/pkg/sinon-esm.js';
+
+describe('createConfig()', () => {
+
+	it('should be exported from index', () => {
+		expect(createConfig).to.equal(createConfigPublic);
+	});
+
+	it('calls WTRConfig.create', () => {
+		const createStub = sinon.stub(WTRConfig.prototype, 'create').callsFake(num => num / 2);
+		const res = createConfig(10);
+
+		sinon.assert.calledOnce(createStub);
+		sinon.assert.calledWith(createStub, 10);
+		expect(res).to.equal(5);
+
+		createStub.restore();
+	});
+});
+
+describe('WTRConfig()', () => {
+	it('should not be exported from index', async() => {
+		const res = (await import('../../src/index.js')).WTRConfig;
+		expect(res).to.be.undefined;
+	});
+});
+
+describe('WTRConfig.create()', () => {
+
+	let config, wtrConfig;
+	before(() => {
+		wtrConfig = new WTRConfig();
+		config = wtrConfig.create();
+	});
+
+	it('should enable nodeResolve', () => {
+		expect(config.nodeResolve).to.be.true;
+	});
+
+	it('should not configure a timeout by default', () => {
+		expect(config.testFramework).to.be.undefined;
+	});
+
+	it('should set timeout when provided with a timeout value', () => {
```

Comment:
Not sure how exhaustive we want to get, but you could test some of the exception handling. (e.g. setting the timeout to a non-Number or the pattern to a non-function throws)